snap/snapcraft.yaml

```diff
@@ -800,100 +781,93 @@ parts:
     stage-packages:
       - perl
     override-build: |
+      cd $SNAPCRAFT_PART_SRC/bundle
+      # apply patches from openresty-kong-patches
+      for i in $SNAPCRAFT_STAGE/openresty-kong-patches/1.13.6.2/*.patch; do patch -p1 < $i; done
```

Comment:
This would be easier to read if you split into multiple lines....

Reply:
Done